kubernetes-examples/portperbroker_plain/postinstall.sh

```diff
@@ -7,7 +7,30 @@
 
 set -eo pipefail
 
+STRIMZI_KAFKA=quay.io/strimzi/kafka:0.35.0-kafka-3.4.0
+KCAT=quay.io/kroxylicious/kcat:1.7.1
+TOPIC=my-topic
+BOOTSTRAP=kroxylicious-service:9292
+
+GREEN='\033[0;32m'
+NOCOLOUR='\033[0m'
+
+echo "Here are some commands to try things out. Run the producer and consumer in separate terminals"
+
+echo -e "${GREEN}Apache Kafka Client${NOCOLOUR}"
+
+echo "To produce to kroxylicious:"
+echo "kubectl -n kafka run java-kafka-producer -ti --image=${STRIMZI_KAFKA} --rm=true --restart=Never -- bin/kafka-console-producer.sh --bootstrap-server ${BOOTSTRAP} --topic ${TOPIC}"
+echo "To consume from kroxylicious:"
+echo "kubectl -n kafka run java-kafka-consumer -ti --image=${STRIMZI_KAFKA} --rm=true --restart=Never -- bin/kafka-console-consumer.sh --bootstrap-server ${BOOTSTRAP} --topic ${TOPIC} --from-beginning"
+
+echo -e "${GREEN}Edenhill Kcat${NOCOLOUR}"
```

Comment:
nit: Is it worth mentioning librdkafka here, as I'd guess its the better known name than `Edenhill`?